registry/polaris/service_discovery.go

```diff
@@ -317,17 +317,20 @@ func (polaris *polarisServiceDiscovery) String() string {
 
 func convertToRegisterInstance(namespace string, instance registry.ServiceInstance) *api.InstanceRegisterRequest {
 
-	health := instance.IsHealthy()
-	isolate := instance.IsEnable()
-	ttl := 5
+	var (
+		health      = instance.IsHealthy()
+		isolate     = instance.IsEnable()
+		ttl         = 5
+		protocolVal = protocolForDubboGO
```

Comment:
What is meaning of `protocolForDubboGO`? If it is refering to the Dubbo procotol, I would suggest using either `const.Dubbo` or `const.DefaultProtocol`, please refer to [const](https://github.com/apache/dubbo-go/blob/3.1/common/constant/default.go).

Reply:
Plus, the protocol is not changed in this case. Is it necessary to declare a `protocolVal` explicitly?

Reply:
```
Protocol: &constant.Dubbo
```

can't pass complier, so need to declare `protocolVal`